apps/admin-gui/src/app/vos/pages/group-detail-page/group-subgroups/group-subgroups.component.ts

```diff
@@ -48,6 +49,14 @@ export class GroupSubgroupsComponent implements OnInit {
     startWith(true)
   );
 
+  roles = new UntypedFormControl();
+  rolesList = ['GROUPADMIN', 'GROUPOBSERVER', 'GROUPMEMBERSHIPMANAGER'];
+  selectedRoles: string[] = [];
+
+  roleTypes = new UntypedFormControl();
```

Comment:
Maybe it can be typed.
```suggestion
  roles = new FormControl(['']);
  rolesList = ['GROUPADMIN', 'GROUPOBSERVER', 'GROUPMEMBERSHIPMANAGER'];
  selectedRoles: string[] = [];

  roleTypes = new FormControl(['']);
```